python/opm/tools/time_vector.py

```diff
@@ -9,65 +9,71 @@
 from opm.io.parser import Parser
 
 # This is from the TimeMap.cpp implementation in opm
-ecl_month = {"JAN" : 1,
-             "FEB" : 2,
-             "MAR" : 3,
-             "APR" : 4,
-             "MAI" : 5,
-             "MAY" : 5,
-             "JUN" : 6,
-             "JLY" : 7,
-             "JUL" : 7,
-             "AUG" : 8,
-             "SEP" : 9,
-             "OCT" : 10,
-             "OKT" : 10,
-             "NOV" : 11,
-             "DEC" : 12,
-             "DES" : 12}
-
-inv_ecl_month = {1 : "JAN",
-                 2 : "FEB",
-                 3 : "MAR",
-                 4 : "APR",
-                 5 : "MAY",
-                 6 : "JUN",
-                 7 : "JUL",
-                 8 : "AUG",
-                 9 : "SEP",
-                 10: "OCT",
-                 11 : "NOV",
-                 12 : "DEC"}
+ecl_month = {
+    "JAN": 1,
+    "FEB": 2,
+    "MAR": 3,
+    "APR": 4,
+    "MAI": 5,
+    "MAY": 5,
+    "JUN": 6,
+    "JLY": 7,
+    "JUL": 7,
+    "AUG": 8,
+    "SEP": 9,
+    "OCT": 10,
+    "OKT": 10,
+    "NOV": 11,
+    "DEC": 12,
+    "DES": 12,
```

Comment:
I know, you fixed only indentation here and this was like this in the original version. But while you are at it: Shouldn't this read "DEC"?

Reply:
> shouldn't this read "DEC"?

You'll notice that there is a `DEC` line just above the `DES` line.  For what it's worth, ECLIPSE ([and Flow](https://github.com/OPM/opm-common/blob/120ef995109afec5757352fc0e30c2c36ac6f172/src/opm/common/utility/TimeService.cpp#L33)) allows a Scandinavian spelling of certain month name abbreviations.  This alternate spelling applies to [MAY](https://github.com/OPM/opm-common/blob/120ef995109afec5757352fc0e30c2c36ac6f172/src/opm/common/utility/TimeService.cpp#L38-L39), [OCT](https://github.com/OPM/opm-common/blob/120ef995109afec5757352fc0e30c2c36ac6f172/src/opm/common/utility/TimeService.cpp#L45-L46), and [DEC](https://github.com/OPM/opm-common/blob/120ef995109afec5757352fc0e30c2c36ac6f172/src/opm/common/utility/TimeService.cpp#L48-L49).

We use the English spelling when translating from month indices to month names, however.